src/components/manager/ManagerDropBoxContent.js

```diff
@@ -121,17 +119,7 @@ const FileDisplay = ({file, tree, treeLoading}) => {
         setFileLoadError("");
 
         (async () => {
-            if (!file) return;
-
-            if (fileExt === "pdf") {
-                setLoadingFileContents(true);
-            }
-
-            if (!textFormat) return;
-
-            if (fileContents.hasOwnProperty(file)) {
-                return;
-            }
+            if (!file || !textFormat || fileContents.hasOwnProperty(file)) return;
```

Comment:
this removes some special pdf handling code

Reply:
I think the removed `setLoadingFileContents(true)` was redundant since it is called in the try block also. So if `fileExt === "pdf"` is true it would be called two times. I didnt notice any change visually either way so I factored it out.

Reply:
pdf doesn't reach the try/catch since it isn't a text-formatted

Reply:
Oh right, fixed it